### ISSUE TITLE
There are never more than two dates concurrent at any instant

### ISSUE THREAD

**Issue by propensive** [davedelong/calendar_fallacies#23]

**Comment by ewjoachim** [comment]
Do you mean 17/01/01 23:50 in, UTC+14, 17/01/02 01:50 in UTC+12 and 17/01/03 01:50 in UTC-12 ?

**Comment by ccoenen** [comment]
I took the liberty of reformatting that ;-)
> Do you mean 2017-01-01 23:50 in, UTC+14, 2017-01-02 01:50 in UTC+12 and 2017-01-03 01:50 in UTC-12 ?

**Comment by ewjoachim** [comment]
Yeah, didn't realize this wasn't ISO 8601, sorry.

https://xkcd.com/1179/

**Comment by propensive** [comment]
Yes, that's the sort of thing I was thinking of - sorry for not providing a more detailed description!
